common/src/main/java/apoc/ApocConfig.java

```diff
@@ -188,30 +188,7 @@ public void init() {
     }
 
     protected String determineNeo4jConfFolder() {
-        String defaultPath = neo4jConfig
-                .get(neo4j_home)
-                .resolve(Config.DEFAULT_CONFIG_DIR_NAME)
-                .toString();
-        String command = System.getProperty(SUN_JAVA_COMMAND);
-        if (command == null) {
-            log.warn(
-                    "system property %s is not set, assuming %s as conf dir. This might cause `apoc.conf` not getting loaded.",
-                    SUN_JAVA_COMMAND, defaultPath);
-            return defaultPath;
-        } else {
-            final String neo4jConfFolder = Stream.of(command.split("--"))
-                    .map(String::trim)
-                    .filter(s -> s.startsWith(CONFIG_DIR))
-                    .map(s -> s.substring(CONFIG_DIR.length()))
-                    .findFirst()
-                    .orElse(defaultPath);
-            if (defaultPath.equals(neo4jConfFolder)) {
-                log.info("cannot determine conf folder from sys property %s, assuming %s", command, defaultPath);
-            } else {
-                log.info("from system properties: NEO4J_CONF=%s", neo4jConfFolder);
-            }
-            return neo4jConfFolder;
-        }
+        return neo4jConfig.get(configuration_directory).toString();
```

Comment:
I can't comment directly on the correct line, but where this is used now has an incorrect comment, can you fix that up? :) also the log info is wrong now

Reply:
Could the log say something like this?
```suggestion
        String neo4jConfFolder = neo4jConfig.get(configuration_directory).toString();
        log.info("from database setting: server.directories.configuration=%s", neo4jConfFolder);
        return neo4jConfFolder;
```

Reply:
Hmm actually, that line is about setting a system property. Do we even need that?

Reply:
I don't think we need the loging any more. Omitting it and updating the comment.